### PR TITLE
add changelog entries

### PR DESCRIPTION
**Pull request by johannes-vogel**
- [x] #137 - cl already added
- [x] #96 - cl already added
- [x] #122
- [x] #120 - cl already added
- [x] #110 - cl already added
- [x] #108 - cl already added
- [x] #107
- [x] #102
- [x] #97 - cl already added
- [x] #95

### CODE REVIEW

**Comment by patricebender** [comment]
I did on purpose not mention changes related to calc elements -> needs some more testing before this gets advertised prominently. Is that okay?

File: postgres/CHANGELOG.md [hunk 1]
```diff
@@ -4,18 +4,25 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version 1.1.0 - tbd
+## Version 1.1.0 - 2023-07-31
 
 ### Added
 
-- connectivity to Azure PostgreSQL
+- Connectivity to Azure PostgreSQL
```
Comment (patricebender) [inline postgres/CHANGELOG.md:11]:
```suggestion
- Connectivity to Azure PostgreSQL.
```

File: sqlite/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,11 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version 1.1.0 - tbd
+## Version 1.1.0 - 2023-07-31
+
+### Changed
+
+- Updated minimum required version of `@cap-js/db-service`
```
Comment (patricebender) [inline sqlite/CHANGELOG.md:11]:
```suggestion
- Updated minimum required version of `@cap-js/db-service` .
```

File: db-service/CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version 1.1.0 - tbd
+## Version 1.1.0 - 2023-07-31
```
Comment (sjvans) [inline db-service/CHANGELOG.md:7]:
```suggestion
## Version 1.1.0 - 2023-08-01
```

File: postgres/CHANGELOG.md [hunk 4]
```diff
@@ -4,18 +4,25 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version 1.1.0 - tbd
+## Version 1.1.0 - 2023-07-31
```
Comment (sjvans) [inline postgres/CHANGELOG.md:7]:
```suggestion
## Version 1.1.0 - 2023-08-01
```

File: sqlite/CHANGELOG.md [hunk 5]
```diff
@@ -4,7 +4,11 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Version 1.1.0 - tbd
+## Version 1.1.0 - 2023-07-31
```
Comment (sjvans) [inline sqlite/CHANGELOG.md:7]:
```suggestion
## Version 1.1.0 - 2023-08-01
```

**Review by sjvans — APPROVED** [review]
🏂

**Pull request merged**